docs/src/experiment_setup_guide.md

```diff
@@ -0,0 +1,115 @@
+`prior = "Parameterized(Normal(5.1393230, 0.225667))"`
+
+You can also ensure that the bounds match the `constraint` entry in your TOML file. 
+Since `constraint` is a TOML string parsed to a vector in Julia, ensure that you have quotes around the square brackets for the constraint list.
+
+Constraint constructors:
+- Lower bound: `bounded_below(0)`
+- Upper bound: `bounded_above(2)`
+- Upper and lower bounds: `bounded(0, 2)`
+
+## Observation Map
+This component is inflexible for now. Your observation map file must be a script with the name `observation_map.jl` which exports a function `observation_map(iteration)`. 
+These requirements are because the observation map is computed in the update step like so:
+```julia
+include("experiments/$job_id/observation_map.jl")
+observation_map(iteration)
+```
+This function must load in model diagnostics for each ensemble member in the iteration and construct an array `arr = Array{Float64}(undef, dims..., ensemble_size)` such that 
+`arr[:, i]` will return the i-th ensemble member's observation map output.
+
+In the update step of EKI, the array will be saved in a JLD2 file named `observation_map.jld2` in the iteration folder of the output directory.
+
+If you are running a perfect-model experiment and generating truth data from ClimaAtmos itself, you may find it useful to create a kernel function to compute the observation map for each ensemble member. You can use this to run the default simulation's diagnostic output through the observation map and save the truth data and noise. An example of this is found in `experiments/sphere_held_suarez_rhoe_equilmoist/observation_map.jl`
+
+## Generating Truth Data
+The truth data must be a vector of observations with the same dimensions as an individual ensemble member entry in the observation map (`arr[:, i]` from above).
+
+The noise should be the covariance matrix of the full data that the truth comes from.
+
+Save each of these to a JLD2 file in the experiment folder. These will be read in when constructing the EKP object.
+
+!!! note
+
+    For full reproducibility, create a script that generates the truth data.
+    If running a perfect-model scenario, the script should run the model and use the resulting diagnostic output to generate the truth data.
+
+## EKP Configuration File
+
+For now, your EKP configuration file must be named `ekp_config.yml`
+The EKP configuration file must contain the following:
+
+- `n_iterations`, the number of iterations to run
+- `ensemble_size`, the ensemble size
+- `prior_path`, the path to the TOML file with the prior parameter distributions
+- `parameter_names`, a list of the long names of the parameters being calibrated 
+- `truth_data`, the truth data
+- `truth_noise`, the covariance of the truth data
+- `output_dir`, the folder where you want calibration data and logs to be output. This must be the same as the `output_dir` in the model configuration file.
+
+## Summary
```

Comment:
I think maybe this should come at the top. 

Also I think the most important thing that is missing is a description of what you actually run which i think is something like "In practice the user calls following 2 scripts:
- `sbatch XYZ generate_observations.sbatch` generates the data for a perfect model experiment defined by `atmos_config.yml`, `observation_map.jl` and `overrides.toml`
- `bash job_array.sh` runs a calibration experiment on this data using HPC, from priors in `prior.toml`, and algorithm setup in `ekp_config.yml`.